nslsii/tests/test_ipynb.py

```diff
@@ -6,3 +6,9 @@ def test_ipynb():
     obj.metadata
     obj.url
     obj.reload()
+
+def test_touchbl():
+    from nslsii.common import if_touch_beamline
```

Comment:
To enable this import, we need to add a line like that in `nsls2/common/__init__.py`:
```py
from .touchbl import if_touch_beamline
```